apps/backend/app/api/auth/register/route.ts

```
@@ -0,0 +1,240 @@
+import { NextResponse } from "next/server";
+import { Gender, PrismaClient, UserType, Branch} from "@prisma/client";
+import bcrypt from "bcryptjs";
+import { UserError, Status, CreateUserRequest} from "app/types/enums";
+import { sanitize } from "class-sanitizer";
+import { escape } from "validator";
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)